spec.html

```diff
@@ -758,12 +758,18 @@ <h1>InitializeTypedArrayFromTypedArray ( _O_, _srcArray_ )</h1>
           1. Let _bufferConstructor_ be ? SpeciesConstructor(_srcData_, %ArrayBuffer%).
         1. Else,
           1. Let _bufferConstructor_ be %ArrayBuffer%.
+        1. <ins>Let _data_ be ? AllocateArrayBuffer(_bufferConstructor_, _byteLength_).</ins>
+        1. <ins>If IsIntegerIndexedObjectOutOfBounds(_srcArray_, _getSrcBufferByteLength_) is *true*, throw a *TypeError* exception.</ins>
```

Comment:
You should create a new getter via MakeIdempotentArrayBufferByteLengthGetter here. The existing _getSrcBufferByteLength_ has cached the length.

Reply:
Oh, of course!